### ISSUE TITLE
redis deprecation warning

### ISSUE THREAD

**Issue by gingerlime** [danmayer/coverband#379]
Hey again 👋 

I bumped into this on the console, and seems related to coverband.

```
`Redis#exists(key)` will return an Integer in redis-rb 4.3. `exists?` returns a boolean, you should use it instead. To opt-in to the new behavior now you can set Redis.exists_returns_integer =  true. To disable this message and keep the current (boolean) behaviour of 'exists' you can set `Redis.exists_returns_integer = false`, but this option will be removed in 5.0. (/var/local/kenhub/kenhub.d/748/vendor/bundle/ruby/2.6.0/gems/coverband-4.2.5/lib/coverband/collectors/view_tracker.rb:111:in `report_views_tracked')
```

Hope to find some time to dig deeper, but thought I'll just report it so it's not forgotten first.

**Comment by danmayer** [comment]
thanks @gingerlime we just fixed this but I haven't done a release

https://github.com/danmayer/coverband/pull/377

trying to decide if I do another 4.x release or just do a release candidate for 5.0

**Comment by gingerlime** [comment]
oooh 5.0 sounds exciting! ✨

**Comment by gingerlime** [comment]
sorry I missed #377 (only checked the open issues, but didn't think to check the closed ones). Will close this since it's a duplicate.

**Comment by danmayer** [comment]
I decided to cut a 4.2.6 release with this redis fix since the 5.0 release still needs some work. @gingerlime

**Comment by gingerlime** [comment]
Thanks for the update @danmayer. I really appreciate it. It's always so nice to work with such a responsive and thoughtful team behind this gem.
